### PR TITLE
Cloudwatch datasource

### PR DESCRIPTION
**Pull request by tkummer33**
### What does this PR do?
This PR adds template for cloudwatch datasource to configure-grafana-datasource playbook.

### How should this be tested?
Run the automation with correct host in the inventory

cloudwatch datasource host
```
---
access_key: "my-cloudwatch-key"
secret_access_key: "my-cloudwatch-secret"
```

prometheus datasource host
```
datasource_url: "https://prometheus-k8s-openshift-monitoring.apps.openshift-1.example.com"
bearer_token: "my-secret-bearer-token"
```

### Is there a relevant Issue open for this?
Provide a link to any open issues that describe the problem you are solving.
resolves #<number>

### Other Relevant info, PRs, etc.
Please provide link to other PRs that may be related (blocking, resolves, etc. etc.)

### People to notify
cc: @redhat-cop/monitoring

### CODE REVIEW

File: grafana/generic/configure-grafana-datasource/tasks/main.yml [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
     force_basic_auth: true
     headers:
       Accept: "application/json"
       Content-type: "application/json"
-  loop: "{{ groups['grafana_datasources'] }}"
+  loop: "{{ groups['grafana_prometheus_datasources'] }}"
+  register: result
+  failed_when:
+    - '"already exists" not in result.json.message'
+    - result.status != 200
```
Comment (pabrahamsson) [inline grafana/generic/configure-grafana-datasource/tasks/main.yml]: I think you might want to use the [`status_code` parameter](https://docs.ansible.com/ansible/latest/modules/uri_module.html#parameter-status_code) to the `uri` module  and allow for both `[200,409]`. That should let you remove the `failed_when`.

**Review by pabrahamsson — CHANGES_REQUESTED** [review]
Looks good @tkummer33, left a comment for handling already existing datasources inline.

File: grafana/generic/configure-grafana-datasource/tasks/main.yml (hunk 1 above)
Comment (tkummer33) [reply grafana/generic/configure-grafana-datasource/tasks/main.yml]: Fixed. Thanks for pointing this out to me!

**Review by pabrahamsson — APPROVED** [review]
LGTM

**Pull request merged**